lib/modules.nix

```diff
@@ -326,7 +332,7 @@ rec {
 
       # Apply the 'apply' function to the merged value. This allows options to
       # yield a value computed from the definitions
-      value = if opt ? apply then opt.apply valueDefined else valueDefined;
```

Comment:
We can drop the `valueDefined` binding above.

Reply:
Nice catch!